staging/src/k8s.io/api/resource/v1beta2/types.go

```diff
@@ -804,7 +804,9 @@ type DeviceSubRequest struct {
 	// +optional
 	AllocationMode DeviceAllocationMode `json:"allocationMode,omitempty" protobuf:"bytes,4,opt,name=allocationMode"`
 
-	// Count is used only when the count mode is "ExactCount". Must be greater than zero.
+	// Count is used only when the count mode is "ExactCount". Must be greater than zero
```

Comment:
```suggestion
	// Count is used only when the AllocationMode is "ExactCount". Must be greater than zero
```

Reply:
I've fixed this in all the API versions.